specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -1237,12 +1237,28 @@
           "description": "Location of the backend (IP address or FQDN)",
           "type": "string"
         },
-        "privateLinkId" : {
-          "description": "If this backend is private, provide the private link resource Id. Populating this optional field indicates that this backend is 'Private'",
+        "privateLinkResourceId": {
```

Comment:
This is a breaking change - did the API change, or was this incorrect as checked in?

Reply:
We've made a few changes to the API since this version was checked in. However, we haven't published this version as of yet, so it's not currently being used.

I've reached out to ARM oncall (Chris Eggert) prior to making these changes to verify.

Reply:
Confirmed: "2020" does not appear in the checked in manifest currently. In the future, you can avoid this by telling the SDK reviewer to not merge the PR or by putting "[DO NOT MERGE]" in the PR title.

---
In reply to: [377863533](https://github.com/Azure/azure-rest-api-specs/pull/8322#discussion_r377863533) [](ancestors = 377863533)